### DISCUSSION TITLE
Add "Open TablePlus" or similar DB tools to menu

### DISCUSSION THREAD

**Discussion by peterkraume** [php-perfect/ddev-intellij-plugin#98] [Ideas]
It would be great to have "Open TablePlus" instead of "Open Database Administration" in the context menu.
Even better would be to have the context menu customiseable, eg. for ddev pull or other ddev commands.

**Comment by nico-loeber** [comment]
Hey @peterkraume,

Sorry I missed your discussion as I wasn't notified by email for new discussions.
I would like to awnswer you anyway:

The context menu is already created dynamically. If there is a DDEV Service running that is providing a URL then it will be listed.
For example if you install a ELK stack you should see "Open Kibana".
The PHPMyAdmin thing is a bit special, as it's currently not an own service but runs in the web container.
Also the type PHPMyAdmin or whatever depends on the database engine. 

Other ddev commands can be added to `Tools > Ddev` menu, that's not a big deal.
As you would like to run ddev get most probably with an argument, the name of the service to get, it would require user input to be run from the GUI. Thats possible of course but I think typing it will be the faster way in this case.

Sorry for the very late reply.
